streams/id.m3u

```diff
@@ -343,3 +343,7 @@ https://ott-balancer.tvri.go.id/live/eds/TVRIWorld/hls/TVRIWorld.m3u8
 https://streamtawaf.asfatv.com/hls/tawaftv.m3u8
 #EXTINF:-1 tvg-id="TVMu.id",TV Mu (720p) [Not 24/7]
 https://e.siar.us/live/tvmu.m3u8
+#EXTINF:-1 tvg-id="",SPOTV (720p)
+http://202.150.161.68/spotv1/index.m3u8
+#EXTINF:-1 tvg-id="",SPOTV2 (720p)
+http://202.150.161.68/spotv2/index.m3u8
```

Comment:
I don't quite understand why you want to move those channels there.

![image](https://github.com/iptv-org/iptv/assets/30985701/33a494be-ffaa-4dbd-8a1c-3f49f784a720)

Reply:
It's an Indonesian feed.